pkg/crmanager/postManager.go

```diff
@@ -309,6 +309,38 @@ func (postMgr *PostManager) GetBigipAS3Version() error {
 	return fmt.Errorf("Error response from BIGIP with status code %v", httpResp.StatusCode)
 }
 
+// GetBigipRegKey ...
+func (postMgr *PostManager) GetBigipRegKey() (string, error) {
+	url := postMgr.getBigipRegKeyURL()
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		log.Errorf("Creating new HTTP request error: %v ", err)
+		return "", err
+	}
+
+	log.Infof("Posting GET BIGIP Reg Key request on %v", url)
```

Comment:
can we remove this log or have it in debug level ?

Reply:
Yes, We can move it to debug level.